src_python/habitat_sim/utils/settings.py

```diff
@@ -30,6 +31,7 @@
     "equirect_semantic_sensor": False,
     "seed": 1,
     "physics_config_file": "data/default.physics_config.json",
+    "enable_physics": True,
```

Comment:
What are the consequences of enabling physics by default here? Do we want to do this?

Reply:
Fair point. This means any Simulator initialized from the default settings util will have access to Bullet API (if installed).
Given that tests are passing, I don't think this is problem. 

Some justification:
1. If user installed bullet compatible Habitat build, they should get Bullet API by default.
2. This incurs very little overhead for static scenes (builds collision meshes, but doesn't do any heavy compute), so it shouldn't reduce performance for non-physics use cases most of the time (e.g. PointNav with HM3D).
3. This is a fairly new utility Dict for quick-start and is not currently used downstream by Habitat-lab (outside of one test for the rearrange task which is physics based), so it is not likely a breaking change.

We could add the breaking change tag to the PR and mention this in the description.
If you think this warrants more discussion I could move this change to its own PR as it is not necessary here.

Reply:
Sounds good. I think we can proceed with it here.

Reply:
@aclegg3 @0mdc What's the verdict on this change? Should we add the breaking change tag?  Otherwise, this PR looks good to merge.

Reply:
I think that it would be preferable to add this change to another PR for traceability. However, I don't feel strongly either way.